src/Manager/UserManager.php

```diff
@@ -183,6 +183,25 @@ public function getOrCreateUserForSignalementAndEmail(Signalement $signalement,
         return null;
     }
 
+    public function getUserCreatorOfSignalement(Signalement $signalement): ?User
+    {
+        $mail = ($signalement->getIsNotOccupant())
+            ? $signalement->getMailDeclarant()
+            : $signalement->getMailOccupant();
+
```

Comment:
A moins que je dise une bêtise, ici on pourrait simplement faire `return $this->findOneBy(['email' => $mail]);` et se passer du reste (mais je ne suis pas sur de la pertinence de cette fonction voir mon autre commentaire)

Reply:
traité